### DISCUSSION TITLE
Leonardo phoenix - open source

### DISCUSSION THREAD

**Discussion by hoodady** [lllyasviel/stable-diffusion-webui-forge#816] [Ideas]
![RDT_20240617_1323105935444427624318155](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/76712106/fd8a8a06-7d62-4335-adef-d532c74d4c56)
I wish this community could find a way to work together and train a model like phoenix.

**Comment by FCB-Sunny** [comment]
Yes! Good idea! I want to train this model! but this is open source? where can I find the model checkpoint?

**Comment by FCB-Sunny** [comment]
I mean this new image generation model - is leonardo phoenix is open-source
model? and if this is right, how can I train?

**Reply by minsky91** [reply]
Leonardo Phoenix is a proprietary model designed by Leonardo.ai and used on their  platform as one of major income sources. Leonardo also offers API for 3rd party clients, Phoenix is one of the core elements of that service, as far as I know. Making the model open source would be for Leonardo.ai like shooting their own foot. Moreover, now that Leonardo has been bought by Canvas, there is a work in progress to train upcoming versions of the Phoenix model on the huge image base of that company. There is just no single point for the company to make it open source.

Besides, there are better open weight models than Phoenix, like Flux, and there are more on the horizon, open source and all.

But, regarding training your own LoRas with Leonardo's tools, it is actually possible, and has been for a while. Should be possible for the Phoenix model soon as well, I guess. You just get a subscription to Leonardo, learn their docs and presto. You won't be able to use/share it outside the Leonardo platform though, it is a proprietary ecosystem, once again.

**Reply by aoleg** [reply]
> Leonardo Phoenix is a proprietary model designed by Leonardo.ai and used on their platform as one of major income sources. 

Then I guess it does not belong here according to the new rules regarding "soft advertising".

**Comment by FCB-Sunny** [comment]
Thanks for your reply. If so, how can I fine tune the leonardo phoenix?
I must log in to the site?
And also, now I wanna publish the docker image for the leonardo phoenix. I
mean I want to build the service by using this API.
If so, how can I do it?
Thanks
FCB-Sunny
